examples/flutter_gallery/lib/demo/material/buttons_demo.dart

```diff
@@ -70,7 +70,7 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
           child: buildRaisedButton(),
         ),
         exampleCodeTag: _raisedCode,
-        documentationUrl: 'https://docs.flutter.io/flutter/material/RaisedButton-class.html',
+        documentationUrl: 'https://api.flutter.dev/flutter/material/RaisedButton-class.html',
```

Comment:
eg. this one is currently a 404. Though if you're not planning on landing this until the domains are swapped over properly, that's probably not an issue.